app-gateway.tf

```diff
@@ -119,7 +119,7 @@ module "appGw" {
       probe                          = "http-probe-gateway"
       PickHostNameFromBackendAddress = "False"
       HostName                       = "${var.external_hostname_gateway}"
-      timeout                        = "${var.documents_request_timeout}"
+      requestTimeout                 = 300
```

Comment:
we need to be extremely careful, we my risk of breaking Prod?
maybe better testing this on another env first? like Perf Test?